helm/alfresco-content-services/values.yaml

```diff
@@ -391,29 +391,123 @@ alfresco-infrastructure:
   nginx-ingress:
     enabled: false
 
-alfresco-search:
+alfrescosearch:
```

Comment:
Can we call this "search" rather than "alfrescosearch" as we don't prefix anything else i.e. repository, share etc.

Reply:
I would leave it as "alfresco-search" and do a rename in a separate PR if indeed required.
The current list of changes is big enough already, renames are making harder to review.

Reply:
Correct, all the other pods are prefixed with "alfresco-cs" so I think naming this one "search" would end up being "alfresco-cs-search" when deployed i.e. consistent. I presume the "alfresco-cs" prefix gets added to all pods somewhere!?

Reply:
Im not sure how easy to change this naming prefix for alfrescosearch, I will check this and come back later

Reply:
Yep, OK, as discussed in the meeting, let's integrate with the minimum number of changes and we can do the rename in a separate ticket later as Alex suggested.

Reply:
@gavincornwell, @killerboot - The name has been changed to "alfresco-search" and the prefix will get override during deployment, so pod full name will be "acs-alfresco-search-solr"